core/src/main/java/com/graphhopper/storage/GraphHopperStorage.java

```diff
@@ -95,42 +94,37 @@ public void freeze() {
         }
     }
 
+    public CHGraph getCHGraph() {
+        Collection<CHGraphImpl> chGraphs = getAllCHGraphs();
+        if (chGraphs.isEmpty()) {
+            throw new IllegalStateException("There is no CHGraph");
+        } else if (chGraphs.size() > 1) {
+            throw new IllegalStateException("There are multiple CHGraphs, use getCHGraph(Weighting) to retrieve a specific one");
```

Comment:
Previously we simply returned the first CHGraph, but I think throwing an exception here is safer so one does not accidentally take the wrong one.